pkg/cluster/internal/providers/docker/provider.go

```diff
@@ -253,3 +253,8 @@ func (p *Provider) CollectLogs(dir string, nodes []nodes.Node) error {
 	errs = append(errs, errors.AggregateConcurrent(fns))
 	return errors.NewAggregate(errs)
 }
+
+// DeleteNetwork does nothing for docker
```

Comment:
We should not do this.
DeleteNodes in podman should clean up the network if no nodes are left.

Reply:
Or the API at provider level should be DeleteCluster (resources only not kubeconfig) and we figure out the per node stuff later.

Reply:
ack